packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/index.js

```diff
@@ -150,23 +132,32 @@ const ListPage = () => {
               onConfirmDeleteAll={deleteAllMutation.mutateAsync}
               onConfirmDelete={(id) => deleteAllMutation.mutateAsync([id])}
               headers={headers}
-              rows={data?.results}
+              rows={users}
               withBulkActions
               withMainAction={canDelete}
             >
               <TableRows
                 canDelete={canDelete}
                 headers={headers}
-                rows={data?.results || []}
+                rows={users}
                 withBulkActions
                 withMainAction={canDelete}
               />
             </DynamicTable>
-            <PaginationFooter pagination={data?.pagination} />
+
+            {pagination && <PaginationFooter pagination={pagination} />}
           </>
         )}
       </ContentLayout>
-      {isModalOpened && <ModalForm onToggle={handleToggle} queryName={queryName} />}
+      {isModalOpened && (
+        <ModalForm
+          onSuccess={async () => {
+            await refetchAdminUsers();
+            await queryClient.refetchQueries(EE_LICENSE_LIMIT_QUERY_KEY);
+          }}
```

Comment:
Personal preference would be to define this in its own `handleSuccess` function besides having it here inline, but up to you.

Reply:
I would prefer to keep it as is in this instance. Which in generally agree it doesn't really "handle" success, because it is not the result of an action, but more a side-effect when rendering the `ModalForm`.